Assignment6Test

```diff
@@ -0,0 +1,119 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+import assignment6.Assignment6;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Stack;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import assignment6.*;
+import java.util.Arrays;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author maria
+ */
+public class Assignment6Test {
+    
+    public Assignment6Test() {
```

Comment:
You don't need to add an empty constructor. This is default for all Java classes without an explicit constructor.

Reply:
true. thanks